rootfs/etc/services.d/supervisor/finish

```diff
@@ -5,4 +5,4 @@
 if { s6-test ${1} -ne 100 }
 if { s6-test ${1} -ne 256 }
 
-redirfd -w 2 /dev/null s6-svscanctl -t /var/run/s6/services
+/run/s6/basedir/bin/halt
```

Comment:
The old code also redirected exit code from what I understand. Is the new line doing the same?

Reply:
It does the same; both don't redirect the exit code. If you want that, you can do it but that also change